packages/fxa-content-server/app/scripts/models/account.js

```diff
@@ -367,6 +367,34 @@ const Account = Backbone.Model.extend(
       });
     },
 
+    /**
+     * Fetches account details from GET /account, for use by the settings views.
+     * Caches its own result, because it's not intended for that endpoint to be
+     * called repeatedly like some of the polling methods are. If you're really,
+     * really sure you need to, pass the `force` option to force a clean request.
+     *
+     * @param {Object} [options]
+     *   @param {Boolean} [options.force] - Ignore any cached results
+     *
+     * @returns {Promise}
+     */
+    settingsData(options = {}) {
```

Comment:
In the @returns documentation, can you leave a blurb about which fields the result contains? The name tells me *where* the data is used, but not what's in it.

Reply:
The trap I wanted to avoid was defining something in a comment here that later becomes a lie when we return more stuff from `GET /account`. It seems unlikely that we (or at least I) will remember to come and update this comment when we do that. So I've hedged and said that it returns all the data from that endpoint, and linked to the module where the endpoint is defined.

Good enough?